internal/kuberesource/sets.go

```diff
@@ -525,7 +525,8 @@ func VolumeStatefulSet() []any {
 							Container().
 								WithName("volume-tester-init").
 								WithImage("ghcr.io/edgelesssys/contrast/initializer:latest").
-								WithCommand("/bin/sh", "-c", CryptsetupInitCommand()).
```

Comment:
Can we replace all `CryptsetupInitCommand` with this subcmd and then remove the script from the codebase?